### ISSUE TITLE
Backend - crud missing people

### ISSUE THREAD

**Issue by vaniaon** [dieegom/brumadinho_location#47]
Create a service that receives the parameters listed in #30 and stores it in a data repository.
This service also needs to provide a list of all missing people

**Comment by odineiramone** [comment]
Can I work on this issue? My idea is to create a Ruby on Rails app that serve and receive this information over a Rest API and store this information in a Postgres database.

**Comment by odineiramone** [comment]
Or, even better: I can create a reporting-service API to concentrate information about missing people [and animals](https://github.com/dieegom/brumadinho_location/issues/49).

I can start to work on missing people CRUD API in Ruby on Rails and help anybody that wants to create the missing animals CRUD inside this app.

**Comment by anapaulagomes** [comment]
Wouldn't be faster creating a Google Forms form and access the information there via API?

**Comment by odineiramone** [comment]
@anapaulagomes, you mean create a Google Form instead create an API to store/access the information?

I've suggested the API's because of these two issues to create an Android app for information about missing [people](https://github.com/dieegom/brumadinho_location/issues/30) and [animals](https://github.com/dieegom/brumadinho_location/issues/48).

But your idea about to create a Google Form is really nice and faster!

**Comment by anapaulagomes** [comment]
Yes, @odineiramone. This way the info would be stored easily and accessing it would be easy too. We won't need to be worried about servers, monitoring etc. We can use the same approach to the other things as well. Time is really important for this project... Building an API, an Android app etc is gonna take a while, IMHO.

**Comment by vaniaon** [comment]
@anapaulagomes  and @odineiramone. We use the photos to try to identify bodies. I do not think google forms allow to send pics. 
Anyway, we can do both to be fast (the service and the form). Just create another issue about it.

**Comment by MattPorto** [comment]
If the rails API is a valid idea I want to help to build it. :)

**Comment by brunolcarli** [comment]
We have the geolocation service with cruds to geopositions, visited  places and foind people.

What if i just add another endpoint with crud to missing people?

Is already implemented service, with no need to build another platform. The frontend can immediatly point to the endpoint and start to insert data.

**Comment by cacarrara** [comment]
Acho que temos que ir por esse caminho nessa aplicação aqui:

- Cadastramento/mapeamento de área de busca
  - Cadastra a área atingida pela catástrofe (geojson?) e exibe em um mapa
- Monitoramento de áreas percorridas
  - Expor uma API para cadastramento da área percorrida com possível alimentação através de mobile apps rastreadores. (lat/lng?)
  - Expõe no mapa da aplicação uma sobreposição entre área de busca e área já percorrida
- Pessoas e vidas
  - Cadastramento de desaparecidos através de uma API (pode ser usado um crawler externo) ou por input manual
  - Registro de pessoas encontradas com vida (API ou manual)
  - Registro de corpos/mortos encontrados (API ou Manual) com ou sem registro de localização do resgate

**Comment by Erick2280** [comment]
> * Monitoramento de áreas percorridas
>   
>   * Expor uma API para cadastramento da área percorrida com possível alimentação através de mobile apps rastreadores. (lat/lng?)
>   * Expõe no mapa da aplicação uma sobreposição entre área de busca e área já percorrida

Atualmente tô trabalhando em um app em Ionic que envia a localização para a API (https://github.com/Erick2280/sosbrumadinholocalizador).

**Comment by VasHX** [comment]
> Acho que temos que ir por esse caminho nessa aplicação aqui:
> 
> * Cadastramento/mapeamento de área de busca
>   
>   * Cadastra a área atingida pela catástrofe (geojson?) e exibe em um mapa
> * Monitoramento de áreas percorridas
>   
>   * Expor uma API para cadastramento da área percorrida com possível alimentação através de mobile apps rastreadores. (lat/lng?)
>   * Expõe no mapa da aplicação uma sobreposição entre área de busca e área já percorrida
> * Pessoas e vidas
>   
>   * Cadastramento de desaparecidos através de uma API (pode ser usado um crawler externo) ou por input manual
>   * Registro de pessoas encontradas com vida (API ou manual)
>   * Registro de corpos/mortos encontrados (API ou Manual) com ou sem registro de localização do resgate

acho que consigo fazer o crawler para manter atualizado a lista de desaparecidos com BS
